targets/targets.json

```diff
@@ -5178,6 +5178,33 @@
             "full", "bare-metal"
         ]
     },
+    "ADAFRUIT_NRF52840_FEATHER": {
+        "supported_form_factors": [
+            "ARDUINO"
+        ],
+        "inherits": [
+            "MCU_NRF52840"
+        ],
+        "macros_add": [
+            "CONFIG_GPIO_AS_PINRESET"
+        ],
+        "extra_labels_remove": [
+            "SOFTDEVICE_NONE"
+        ],
+        "extra_labels_add": [
+            "SOFTDEVICE_COMMON",
+            "SOFTDEVICE_S140_FULL"
+        ],
```

Comment:
As of Mbed OS 6 we only support Cordio as the BLE stack - other implementations such as SoftDevice are not compatible with the current BLE API in Mbed OS.
```suggestion
```
If this target has the same BLE chip as NRF52840_DK, it should probably work - feel free to test with the default Cordio stack.